app/models/mentee_questionnaire.rb

```diff
@@ -1,11 +1,15 @@
 class MenteeQuestionnaire < ApplicationRecord
+  include ULID::Rails
+  ulid :id, auto_generate: true
+  ulid :respondent_id
+
   belongs_to :respondent, class_name: "User", foreign_key: "respondent_id"
 
   validates :respondent_id, presence: true
   validates :name, presence: true
-  validates :currently_writing_ruby, presence: true
+  validates :currently_writing_ruby, inclusion: [true, false]
```

Comment:
Same as MentorQuestionnaire.

https://github.com/goodscary/firstrubyfriend/blob/03fe4ddedae42c53130f6718d11535c7a34cd830/app/models/mentor_questionnaire.rb#L11

Validations wouldn't allow "false" to get persisted to the db.